src/engine/positionscratchcontroller.cpp

```diff
@@ -274,8 +274,8 @@ double PositionScratchController::getRate() {
     return m_dRate;
 }
 
-void PositionScratchController::notifySeek(double currentSample) {
+void PositionScratchController::notifySeek(mixxx::audio::FramePos position) {
     // scratching continues after seek due to calculating the relative distance traveled
     // in m_dPositionDeltaSum
-    m_dLastPlaypos = currentSample;
+    m_dLastPlaypos = position.toEngineSamplePos();
```

Comment:
No validation?

Reply:
No. I consider calling notifySeek with an invalid frame pos a bug. I'll add a debug assert at the start of the method to make this more obvious.